public_html/wp-content/mu-plugins/blocks/assets/src/organizers/organizers-select.js

```diff
@@ -47,9 +47,7 @@ class OrganizersSelect extends Component {
 					};
 				} );
 
-				if ( this.isStillMounted ) {
-					this.setState( { wcb_organizer: posts } );
-				}
+				this.setState( { wcb_organizer: posts } );
```

Comment:
Is there a reason we're using `setState()` here but `this.state = ` in `fetchOrganizerDetails()`?

Reply:
Oh, nevermind, `fetchSelectOptions()` is called synchronously, but the `setState()` call is within a promise callback closure :)